### ISSUE TITLE
Fail to recognize image format

### ISSUE THREAD

**Issue by sauerj** [jellyfin/jellyfin#490]
Just saw a bunch of these errors in the log, not sure if there were there in a previous version.

```
[2019-01-07 16:43:34.377 +00:00] [INF] Getting image size for item "Movie" "/config/metadata/library/27/27ba816802def3965a7b625f384a1181/poster.jpg"
[2019-01-07 16:43:34.378 +00:00] [ERR] Failed to determine primary image aspect ratio for "/config/metadata/library/27/27ba816802def3965a7b625f384a1181/poster.jpg"
System.ArgumentException: Could not recognize image format.
   at Emby.Drawing.Common.ImageHeader.DecodeJfif(BinaryReader binaryReader) in /repo/Emby.Drawing/Common/ImageHeader.cs:line 214
   at Emby.Drawing.Common.ImageHeader.GetDimensions(BinaryReader binaryReader) in /repo/Emby.Drawing/Common/ImageHeader.cs:line 97
   at Emby.Drawing.Common.ImageHeader.GetDimensions(String path, ILogger logger, IFileSystem fileSystem) in /repo/Emby.Drawing/Common/ImageHeader.cs:line 68
   at Emby.Drawing.ImageProcessor.GetImageSize(String path, Boolean allowSlowMethod) in /repo/Emby.Drawing/ImageProcessor.cs:line 499
   at Emby.Drawing.ImageProcessor.GetImageSize(BaseItem item, ItemImageInfo info, Boolean allowSlowMethods, Boolean updateItem) in /repo/Emby.Drawing/ImageProcessor.cs:line 469
   at Emby.Server.Implementations.Dto.DtoService.GetPrimaryImageAspectRatio(BaseItem item) in /repo/Emby.Server.Implementations/Dto/DtoService.cs:line 1445
```

**Comment by sparky8251** [comment]
If you could confirm whether or not they are from an earlier version, that would be nice.

We did make changes to the library that manages images and if you use docker, it was flat out broken for a bit.

**Comment by cvium** [comment]
Fixed in #500 and #501

**Comment by sauerj** [comment]
@hawken93 I can confirm your fix. Do not see the errors anymore.
